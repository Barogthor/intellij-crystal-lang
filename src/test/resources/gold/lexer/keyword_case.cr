case exp
when value1, value2
  do_something
when value3
  do_something_else
else
  do_another_thing
end