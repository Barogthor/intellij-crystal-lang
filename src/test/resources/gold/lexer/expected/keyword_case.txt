case ('case')
WHITE_SPACE (' ')
IDENT ('exp')
WHITE_SPACE ('\n')
when ('when')
WHITE_SPACE (' ')
IDENT ('value1')
, (',')
WHITE_SPACE (' ')
IDENT ('value2')
WHITE_SPACE ('\n  ')
IDENT ('do_something')
WHITE_SPACE ('\n')
when ('when')
WHITE_SPACE (' ')
IDENT ('value3')
WHITE_SPACE ('\n  ')
IDENT ('do_something_else')
WHITE_SPACE ('\n')
else ('else')
WHITE_SPACE ('\n  ')
IDENT ('do_another_thing')
WHITE_SPACE ('\n')
end ('end')